src/Apps/W1/Quality Management/Documentation/qms-lot-blocking-unblocking.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Lot blocking and unblocking
+description: Learn how to block and unblock inventory lots using workflows and grade-specific controls to ensure quality compliance.
+author: brentholtorf
+ms.author: bholtorf
+ms.reviewer: bholtorf
+ms.topic: how-to
+ms.search.form: 
+ms.date: 10/20/2025
+ms.service: dynamics-365-business-central
+ms.custom: bap-template
+
+---
+
+# Lot blocking and unblocking
+
+This article explains how to automatically block and unblock inventory lots based on quality inspection test results using workflows and grade-specific controls. There are two main approaches for lot blocking:
+
+- **Workflow-Based Blocking**: Block lots using workflows.
+- **Grade-Based Blocking**: Document-specific blocking based on test grades
+
+Both approaches help ensure that noncompliant inventory isn't used inappropriately, while allowing flexible quality control processes.
+
+## Prerequisites
+
+- Enable **Workflow integration** in **Quality Inspection Setup**.
```

Comment:
The correct page caption is "[Quality Management Setup](https://github.com/microsoft/BCApps/blob/main/src/Apps/W1/Quality%20Management/app/src/Setup/Setup/QltyManagementSetup.Page.al)". Please fix the name also for all other occurrences of Quality Inspection Setup.